### ISSUE TITLE
Could not find `version` in `ash`

### ISSUE THREAD

**Issue by MoinSebi** [MaikKlein/ash#467]
Hello!
ash-window is a dependency for a tool I tried to install and I get the following error: 

```
     Running `rustc --crate-name ash_window --edition=2018 /home/svorbrugg_local/.cargo/registry/src/github.com-1ecc6299db9ec823/ash-window-0.6.0/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts --crate-type lib --emit=dep-info,metadata,link -C opt-level=3 -C embed-bitcode=no -C metadata=f582c22f5b4b1b49 -C extra-filename=-f582c22f5b4b1b49 --out-dir /home/svorbrugg_local/gfaestus/target/release/deps -L dependency=/home/svorbrugg_local/gfaestus/target/release/deps --extern ash=/home/svorbrugg_local/gfaestus/target/release/deps/libash-720f352fdd135824.rmeta --extern raw_window_handle=/home/svorbrugg_local/gfaestus/target/release/deps/libraw_window_handle-6ef3c679fdcdbb8d.rmeta --cap-lints allow`
error[E0432]: unresolved import `ash::version`
   --> /home/svorbrugg_local/.cargo/registry/src/github.com-1ecc6299db9ec823/vk-mem-0.2.2/src/lib.rs:753:18
    |
753 |         use ash::version::{DeviceV1_0, DeviceV1_1, InstanceV1_0};
    |                  ^^^^^^^ could not find `version` in `ash`

```

I have not found any advice on how to fix this and hoped anyone here can help me :)

**Comment by MaikKlein** [comment]
Hey sorry about that, just delete the import and it should compile. The version traits have been deleted, and now methods on Instance/Device.

**Comment by MarijnS95** [comment]
Hello!

We recently removed these traits in the 0.33 release, and unfortunately vk-mem uses a non-semver requirement: https://github.com/gwihlidal/vk-mem-rs/blob/ec2772294fcded340d663487503fd356b70bc3a3/Cargo.toml#L32

This allows vk-mem to use **any** Ash release beyond 0.27, even though their semver bump allows breaking changes!

This has to be resolved inside `vk-mem`, where it has already been reported: https://github.com/gwihlidal/vk-mem-rs/issues/54. Closing this issue as we have nothing to track for it.
